graspy/pipeline/__init__.py

```diff
@@ -1,3 +1,4 @@
+from .anomaly import anomaly_detection
 from .mug2vec import mug2vec
 
-__all__ = ["mug2vec"]
+__all__ = ["anomaly_detection", "mug2vec"]
```

Comment:
@dwaynepryce I know at one point we discussed putting `mug2vec` in embed and removing the `pipeline` module. Could still do that and just make a new module for this anomaly detection stuff (like `.timeseries` or `.anomaly`), not sure what makes the most sense though

Reply:
Hrm, this is a good question.  I'm still trying to get a handle of the main categories of functionality  we're trying to implement.  In general we have two main categories in one dimension and then cross-cutting categories: single graph operations and multiple graph operations.

And when it comes to multiple graph operations, what kinds do we have?  Timeseries is definitely one, but I can't think of another plausible combination of a plurality of graphs.  But maybe just naming it `.timeseries` lets us semi-future-proof it in the sense that if we ever find another way of thinking of many graphs being operated on in combination but aren't timeseries based, we can.

All of that is to say "I like timeseries, and mug2vec should go in embed"

Reply:
Ill make an issue to move mug2vec to embed since thats outside the scope of this PR